src/util/RedisKvs.ts

```diff
@@ -156,6 +157,7 @@ export class RedisKvs<T> {
           reject(err);
         } else {
           resolve(val);
+          logger.log(`Total Articles stored in redis ${val}`);
```

Comment:
`RedisKvs.len()` is being used not only for articles but has number of usages across the code (for example, for the files), while the original issue is about articles (and also the message logged). I think would be better to move the logging out of this class and put to the outer level.
Another reason to do that would be to keep RedisKvs class low coupled with application logic as possible :)

Reply:
Done.